csrankings.csv

```diff
@@ -13302,4 +13302,6 @@ Yubao Wu,Georgia State University,http://cs.gsu.edu/profile/yubao-wu/,Ussl738AAA
 Zhisheng Yan,Georgia State University,https://grid.cs.gsu.edu/~zyan/,i6ePuTMAAAAJ
 Alex Zelikovsky,Georgia State University,http://cs.gsu.edu/profile/alex-zelikovsky/,UzCRNJwAAAAJ
 Yanqing Zhang,Georgia State University,https://grid.cs.gsu.edu/zhang/,NOSCHOLARPAGE
-Ying Zhu,Georgia State University,https://sites.google.com/site/yingzhugsu/,sz_Z3igAAAAJ
\ No newline at end of file
+Ying Zhu,Georgia State University,https://sites.google.com/site/yingzhugsu/,sz_Z3igAAAAJ
++Kyunghyun Cho,New York University,http://www.kyunghyuncho.me/,0RAmmIAAAAAJ^M
```

Comment:
Spurious "+" here and in the next entry.

Reply:
This person is at Facebook. Half-time employed faculty are not eligible for inclusion.
http://www.kyunghyuncho.me/

Reply:
He is consulting at facebook, 1 day a week.  His NYU position is full time

Reply:
Great, thanks for the clarification (the number of students, etc. is immaterial -- it's all about the full-time appointment). 1 day a week is fine. The spurious "+" signs still need to be fixed.

Reply:
removed spurious "+",  added new faculty